### DISCUSSION TITLE
Cobbler Web

### DISCUSSION THREAD

**Discussion by asimat** [cobbler/cobbler-web#234] [General]
### Is your feature request related to a problem? No

I would like to know if there is any progress with Cobbler-Web ? Is there a working version that can be used with cobbler 3.3.4 ?

### Provide a detailed description of the proposed feature

In my Production environment we still use Cobbler 3.2 as our Juniors find it easy to add systems and assign to groups etc using Cobbler-web.  Ansible can be used to get dynamic inventory from cobbler and puppet also works well. However its time for us to reinstall and upgrade all CentOS 7 servers, so would be good if we could upgrade cobbler and cobbler-web.

### Alternatives you've considered

Moving away from cobbler is not ideal for us, but that is the only alternate is there is no GUI for cobbler

**Comment by SchoolGuy** [comment]
As the UI has its own repository I'll convert this into a GH discussion.

Now in regard to your question: At the moment there is no UI due to a lack of development power for this. I am trying to recruit people for it so we get the manpower but beer is not a very good motivator for most people I know. The basic skeleton of a UI is there already but it is not functional since the API client and the frontend are not connected for the purpose of showing data. Login to a Cobbler server already works.

I am pushing out 3.3.5 in the coming days with a fix so that the logs can finally be retrieved again via the XML-RPC API. Then it will make sense to prioritize the UI a bit higher for myself. However, my time is limited of course so don't expect the UI to be fully working by the end of the month. It is a big piece of code and the weekends are short for projects like this.

If there is a business need for the UI and you are a SUSE customer (or want to become one) you may get in contact with my work mail address (egotthold@suse.com), so I can connect you to somebody who takes care of the business side of things. So I then can follow up on prioritizing the WebUI over the things that are currently on my list at work.
